crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -357,7 +357,7 @@ impl Context {
         while let Some(unresolved_fn_label) = entry_point.first_unresolved_function_call() {
             let artifact = &brillig
                 .find_by_function_label(unresolved_fn_label.clone())
-                .expect("Cannot find linked fn {unresolved_fn_label}");
```

Comment:
Was this change mostly stylistic?

Reply:
oh it's just that this will print `"Cannot find linked fn {unresolved_fn_label}"` literally, without interpolating the value :D

Reply:
Ahh!